docs/content/tutorials/observability.md

```diff
@@ -36,7 +36,7 @@ Install NGINX Service Mesh and configure it to integrate with the observability
 Using the CLI:
 
 ```bash
-nginx-meshctl deploy --prometheus-address "prometheus.nsm-monitoring.svc:9090" --telemetry-exporters "type=otlp,host=otel-collector.nsm-monitoring.svc,port=4317" --telemetry-sampler-ratio 1 --disabled-namespaces "nsm-monitoring"
+nginx-meshctl deploy --prometheus-address "prometheus.nsm-monitoring.svc:9090" --telemetry-exporters "type=otlp,host=otel-collector.nsm-monitoring.svc,port=4317" --telemetry-sampler-ratio 1 --disable-auto-inject --enabled-namespaces "nginx-mesh"
```

Comment:
We don't want to enable injection the nginx-mesh namespace; that's where our control plane lives.